hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -642,6 +642,7 @@ private FlinkOptions() {
   public static final String TIME_ELAPSED = "time_elapsed";
   public static final String NUM_AND_TIME = "num_and_time";
   public static final String NUM_OR_TIME = "num_or_time";
+  public static final String NUM_COMMITS_AFTER_LAST_REQUEST = "num_commits_after_last_request";
   @AdvancedConfig
   public static final ConfigOption<String> COMPACTION_TRIGGER_STRATEGY = ConfigOptions
```

Comment:
The varialbe may not be used anywhere, maybe we should just fix the doc.

Reply:
@danny0405 Thank you for your reply. 

By adding this parameter `'compaction.trigger.strategy'='num_commits_after_last_request'`, in my test, the job can generate a compaction plan for every fixed number of `deltacommits`.

I don't understand that "The variable may not be used anywhere". Is there a problem with this compaction trigger strategy in some scenarios?

Reply:
`NUM_COMMITS_AFTER_LAST_REQUEST` this variable is not used by any other code.

Reply:
Got it.